### PR TITLE
Update library description

### PR DESCRIPTION
**Pull request by jaeklim**

### CODE REVIEW

File: PLCrashReporter.podspec [hunk 1]
```diff
@@ -1,14 +1,15 @@
 Pod::Spec.new do |spec|
-  spec.name       = 'PLCrashReporter'
-  spec.version    = '1.4.0'
-  spec.summary    = 'Reliable, open-source crash reporting for iOS, macOS and tvOS.'
+  spec.name        = 'PLCrashReporter'
+  spec.version     = '1.4.0'
+  spec.summary     = 'Reliable, open-source crash reporting for iOS, macOS and tvOS.'
+  spec.description = 'PLCrashReporter is a reliable open source library that provides an in-process live crash reporting framework for use on iOS, macOS and tvOS. The library detects crashes and generates reports to help your investigation and troubleshooting with the information of application, system, process, thread, etc. as well as stack traces.'
```
Comment (MatkovIvan) [inline PLCrashReporter.podspec]: Let's use multi-line string syntax here

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # PLCrashReporter
 
-Reliable, open-source crash reporting for iOS, Mac OS X and tvOS.
```
Comment (MatkovIvan) [inline README.md]: Should we keep short summary?

Comment (jaeklim) [reply README.md]: `summary` is a required property.

File: PLCrashReporter.podspec (hunk 1 above)
Comment (jaeklim) [reply PLCrashReporter.podspec]: Why? This is just one line.

File: README.md (hunk 2 above)
Comment (jaeklim) [reply README.md]: Do you really want to keep this? This summary is just meaningless and description repeats what it says there.

**Pull request merged**